[2024-08-16T07:16:44.375+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: compare_data2threshold.compare_data2threshold_task manual__2024-08-16T07:16:36.692775+00:00 [queued]>
[2024-08-16T07:16:44.388+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: compare_data2threshold.compare_data2threshold_task manual__2024-08-16T07:16:36.692775+00:00 [queued]>
[2024-08-16T07:16:44.389+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-08-16T07:16:44.409+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): compare_data2threshold_task> on 2024-08-16 07:16:36.692775+00:00
[2024-08-16T07:16:44.417+0000] {standard_task_runner.py:60} INFO - Started process 668 to run task
[2024-08-16T07:16:44.422+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'compare_data2threshold', 'compare_data2threshold_task', 'manual__2024-08-16T07:16:36.692775+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/compare_data2threshold.py', '--cfg-path', '/tmp/tmpga5h8f_c']
[2024-08-16T07:16:44.424+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask compare_data2threshold_task
[2024-08-16T07:16:44.445+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-16T07:16:44.523+0000] {task_command.py:423} INFO - Running <TaskInstance: compare_data2threshold.compare_data2threshold_task manual__2024-08-16T07:16:36.692775+00:00 [running]> on host 282fa7a8e0f3
[2024-08-16T07:16:44.726+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='compare_data2threshold' AIRFLOW_CTX_TASK_ID='compare_data2threshold_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T07:16:36.692775+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-16T07:16:36.692775+00:00'
[2024-08-16T07:16:44.728+0000] {compare_data2threshold.py:36} INFO - unit_threshold : 120
[2024-08-16T07:16:44.729+0000] {compare_data2threshold.py:37} INFO - global_threshold : 2000
[2024-08-16T07:16:44.792+0000] {compare_data2threshold.py:54} INFO - Total des counts : 1479
[2024-08-16T07:16:44.792+0000] {compare_data2threshold.py:60} INFO - Le total 1479 ne d√©passe pas le seuil global 2000
[2024-08-16T07:16:44.795+0000] {compare_data2threshold.py:65} INFO - Au moins une ligne est au-dessus de unit_threshold
[2024-08-16T07:16:44.796+0000] {compare_data2threshold.py:70} INFO - Nous devons reentrainer le model avec les nouvelles data
[2024-08-16T07:16:44.796+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-16T07:16:44.808+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=compare_data2threshold, task_id=compare_data2threshold_task, execution_date=20240816T071636, start_date=20240816T071644, end_date=20240816T071644
[2024-08-16T07:16:44.839+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-16T07:16:44.865+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
